source/connect.txt

```diff
@@ -6,15 +6,85 @@ Connect to Your MongoDB Deployment
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta:: 
+   :description: Connect to your MongoDB deployment with the MongoDB for IntelliJ Plugin.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44809.
+This task page shows you how to connect to your MongoDB deployment using
+the {+intellij-full+}.
+
+Prerequisites
+-------------
+
+Ensure that an instance of MongoDB is running and the {+intellij-short+}
+is installed.
+
+Steps
+-----
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Show the Database tab.
+
+      To show the :guilabel:`Database` tab, click the Database Explorer 
+      icon on the far right menu bar of IntelliJ.
```

Comment:
[nit] Here, "far right" should be hyphenated!
```suggestion
      icon on the far-right menu bar of IntelliJ.
```

Reply:
Updated!